content/ita/careers.md

```diff
@@ -12,17 +12,22 @@ twitterlargeimage: "/images/careers/banner.png"
 
 ![Facile.it - lavora con noi!](/images/careers/banner.png)
 
-**Facile.it** è in costante crescita e siamo sempre alla ricerca di programmatori da aggiungere al nostro team. Questa proposta è rivolta a sviluppatori di grande esperienza, che credono nell’innovazione e sono sempre attenti alle nuove tecnologie e non vedono l’ora di metterle in pratica. Siamo un gruppo di lavoro dinamico, giovane e divertente. Siamo in totale quasi 80 programmatori e gestiamo numerosi progetti con differenti stack tecnologici.
+**Facile.it** is constantly growing and we are always looking for programmers to join our team.
```

Comment:
I'm agree with you, maybe we can use a more generic name for all IT roles.
How about replacing `we are always looking for programmers` with "we are always looking for experts to join in our IT Area"? 
Do you have any suggestions?

Reply:
I'm not sure if the HR department would agree on the term "expert" since we also hire juniors. If we want to use the term expert we should probably check with them
What about `we are always looking for IT professionals to join our team`?